### PR TITLE
DefaultArgParser.py: Update coala help

### PR DESCRIPTION
**Pull request by atultherajput**
Replace "--filter-by-language" with "--filter-by"
on help page because "--filter-by-language" is
deprecated.

Closes https://github.com/coala/coala/issues/4652

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [X] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [X] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: coalib/parsing/DefaultArgParser.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def default_arg_parser(formatter_class=None):
 To find out what kind of analysis coala offers for the languages you use, visit
 http://coala.io/languages, or run::
 
-    $ coala --show-bears --filter-by-language C Python
+    $ coala --show-bears --filter-by C Python
```
Comment (shreyans800755) [inline coalib/parsing/DefaultArgParser.py]: Replace `--filter-by-language` -> `--filter-by language` and not just `--filter-by`

**Comment by shreyans800755** [comment]
@atultherajput 
Also update the description in git commit accordingly.

**Comment by atultherajput** [comment]
@shreyans800755 
Done but now this PR have 2 commits, any problem with that?

**Comment by atultherajput** [comment]
Done, merged both commit :)

**Comment by shreyans800755** [comment]
@atultherajput Changes looks good.
Just small request regarding commit description:
Remove the word `page` as its part of CLI output and not a page.

You may put something like:
`on help page` -> `in coala CLI help string`

**Comment by atultherajput** [comment]
@shreyans800755 
I had changed commit description as you requested. Please review it again.

**Comment by shreyans800755** [comment]
@atultherajput 
Great.

ack d92df23

Not sure why travis is failing. Try to run it again, if the same error persists, ask someone on gitter to have a look.

**Comment by atultherajput** [comment]
@shreyans800755
I am getting same build error again. I think there is some problem in check_setuptools.py file. 
BTW Could you please Merge this PR?

**Comment by atultherajput** [comment]
@sils @Makman2 
Guys this PR is pending due to this issue https://github.com/coala/coala/issues/4708 (Travis CI build always got failed). Now what should I do?

**Comment by shreyans800755** [comment]
@atultherajput 
Sorry, I don't have permission to merge the PR.

**Comment by Makman2** [comment]
@atultherajput yeah looks like you've hit the same error. I'm acking your commit, as it's good, however a merge has to wait until the bug is resolved :+1:

**Comment by Makman2** [comment]
ack d92df23

**Comment by Makman2** [comment]
(Oh it was acked before, however doesn't harm :3)

**Comment by atultherajput** [comment]
@Makman2 I think now the bug is resolved so this PR should merged!

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by atultherajput** [comment]
@Makman2 Please merge this.

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by atultherajput** [comment]
@Makman2 Rebase process is completed.

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by Makman2** [comment]
@rultor merge

**Pull request merged**